pkg/builder/azure/builder.go

```diff
@@ -76,11 +76,25 @@ func (b *Builder) Build(ctx context.Context, app *builder.AppContext, out chan<-
 			imageNames = append(imageNames, fmt.Sprintf("%s:%s", app.Ctx.Env.Name, imageNameParts[len(imageNameParts)-1]))
 		}
 
+		var args []containerregistry.BuildArgument
+
+		// TODO: once the API includes this as default, remove it
+		buildArgType := "DockerBuildArgument"
+		for k := range app.Ctx.Env.ImageBuildArgs {
+			name := k
+			value := app.Ctx.Env.ImageBuildArgs[k]
```

Comment:
I think this can be removed if we use the form `for k, value := range app.Ctx.Env.ImageBuildArgs {`. Would that work?

Reply:
Not necessarily - since both Docker and ACR expect a pointer to a string, if we pass `&v` from `for k, v := range ...`, then it doesn't take the pointer to the actual loop value, but to the loop variable.

The same is valid for your other comment in this PR - the alternative would be to use `to.StringPtr(value)` - using the `autorest` method - it's just a matter of preference that I wanted to do it without an external package.

Reply:
aw dang yeah I missed that. Thanks for pointing that out! :)